src/tests/ftest/util/data_mover_test_base.py

```diff
@@ -388,10 +394,10 @@ def parse_create_cont_uuid(self, output):
             str: The parsed uuid.
 
         """
-        uuid_search = re.search(r"Successfully created container (.*-.*-.*-.*-.*)", output)
-        if not uuid_search:
+        label_search = re.search(r"Successfully created container (.*)", output)
+        if not label_search:
             self.fail("Failed to parse container uuid")
```

Comment:
> self.fail("Failed to parse container uuid")

self.fail("Failed to parse container label")

Reply:
Ah, thanks!